### PR TITLE
Fix DShot documentation styling

### PR DESCRIPTION
**Pull request by KarateBrot**
Makes resources look nice in both color themes (and restructuring of source files)

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

File: docs/development/API/Dshot.mdx [hunk 1]
```diff
@@ -43,7 +50,9 @@ SSSSSSSSSSSTCCCC
 
 The interesting part is, that 1 and 0 in the DShot frame are distinguished by their high time. This means that every bit has a certain (constant) length, and the length of the high part of the bit dictates if a 1 or 0 is being received.
 
-![DShot high times](./dshot_bit.webp)
+<p align="center">
+  <img src={Bit} alt="DShot Bit" className="themeInvert" />
+</p>
```
Comment (haslinghuis) [inline docs/development/API/Dshot.mdx:55]: @VitroidFPV could we retain using markup instead of falling back using html with styling in content?

**Review by haslinghuis — APPROVED** [review]

File: docs/development/API/Dshot.mdx (hunk 1 above)
Comment (VitroidFPV) [reply docs/development/API/Dshot.mdx:55]: This should work as a compromise I think:
```
<p align="center themeInvert">

    ![DShot 8bit](<URL>)

</p>
```
It's possible to put Markdown elements inside HTML elements, but there isn't a Markdown element to specifically center content AFAIK

**Comment by KarateBrot** [comment]
> @nerdCopter 
> i use light theme, but in dark theme i see some images with black on dark background. if this PR is not meant to changes this, then okay.

Weird, on my end it works properly (when compiling with `npm start`). Maybe the deployment server was last updated before the other PR https://github.com/betaflight/betaflight.com/pull/362 was merged.

grafik

File: docs/development/API/Dshot.mdx (hunk 1 above)
Comment (KarateBrot) [reply docs/development/API/Dshot.mdx:55]: Unfortunately that didn't work for me. But maybe I'm just inept, so I could use some help 😅

**Comment by haslinghuis** [comment]
@KarateBrot you are right:

This PR:

image

PR rebased on master:

image

**Pull request merged**